src/components/Filter/Category/category.css

```
@@ -0,0 +1,11 @@
+.container-wraper h2 {
+  font-family: Hind, sans-serif;
+}
+
+.category-menu-header {
+  font-family: Hind, sans-serif;
+}
```

```suggestion
.font-hind {
  font-family: Hind, sans-serif;
}
```
Chỗ này e css 1 cái css font-hind để tái sử dụng thì hay hơn